01-python-basics/hw-01-python-basics.py

```diff
@@ -0,0 +1,51 @@
+# import lib with needed functions
+import random
+
+# create list of 100 random numbers from 0 to 1000
+# using sample method that returns a list of 100 elements in range between 0 and 1000
+random_list = random.sample(range(0, 1000), 100)
+
+# sort list from min to max (without using sort())
+# go through each element of list
+for i in range(len(random_list)):
+    # add another loop for comparison 2 neighboring elements
+    for j in range(len(random_list) - i - 1):
+        # compare elements
+        if random_list[j] > random_list[j + 1]:
+            # swap elements is they are not in order
+            # add temp variable for storing element with higher value that will be swapped
```

Comment:
changing values for two items can be done in one line:

random_list[i], random_list[j] = random_list[j], random_list[i]